src/main/kotlin/com/openlattice/linking/controllers/RealtimeLinkingController.kt

```diff
@@ -52,6 +56,26 @@ class RealtimeLinkingController(
         return linkableEntitySets.minus(entitySetsNeedLinking)
     }
 
+    @RequestMapping(
+            path = [RealtimeLinkingApi.MISSING],
+            method = [RequestMethod.POST],
+            produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+
+    override fun getEntitySetNotYetLinkedCount(@RequestBody entitySetIds: Set<UUID>): Map<UUID, Long> {
+        ensureAdminAccess()
+        val linkableEntitySets = lqs
+                .getLinkableEntitySets(linkableTypes, entitySetBlacklist, whitelist.orElse(setOf()))
+                .toSet()
+        val linkableRequestedEntitySets = linkableEntitySets
+                .intersect(entitySetIds)
+        val nonLinkableRequestedEntitySets = entitySetIds.subtract(linkableEntitySets)
```

Comment:
What is `nonLinkableRequestedEntitySets` used for?

Reply:
it's removed